distr_tracing/distr_tracing_tempo/distr-tracing-tempo-configuring.adoc

```diff
@@ -17,6 +17,11 @@ endif::[]
 
 include::modules/distr-tracing-tempo-config-default.adoc[leveloffset=+2]
 
+[role="_additional-resources"]
+[id="additional-resources_distr-tracing-tempo-configuring"]
+===== Additional resources
```

Comment:
H5? Is this misaligned?

Reply:
Thank you. Checking.

Reply:
As I suspected, removing this didn't solve the issue. The jobs are still stuck. See below.